crypto/src/sqr.s

```diff
@@ -2,6 +2,17 @@
 //
 // public domain
 //
+//// This assmeble file was taken from
+// https://munacl.cryptojedi.org/curve25519-cortexm0.shtml
+// Credit to  Björn Haase and Ana Helena Sánchez, as well
+// as M. Hutter and P. Schwabe
+// This assemble file assumes the function multiply256x256_asm
```

Comment:
"exports" or "exposes" not "assumes". "assumes" implies there is such a function defined somewhere else and this file relies on that function.